content/documentation/overview/content.md

```diff
@@ -24,7 +24,7 @@ To make your life easier, you don't have to implement all of those Model Hub cap
 Instead EMF Cloud provides reusable libraries and integration code for third-party components to cover the most common choices and formats, such as JSON files.
 Also, EMF Cloud contains libraries that make it easy to connect and interact with the Model Hub.
 
-➡️ Best to [get started]({{< relref  "gettingStarted" >}}) with the Coffee Editor NG!
+➡️ Best to [get started]({{< relref  "gettingStarted" >}}) with one of our example applications: Model Hub examples for model management or CrossModel for DSL support.
```

Comment:
I think it is probably best to lead the reader first to the "Modeling Tool Approaches" page and then from there to two distinct getting started guides.